### ISSUE TITLE
New vision/eye phenotype FO-wave

### ISSUE THREAD

**Issue by abby-jonesKOMP** [obophenotype/mammalian-phenotype-ontology#3344]
>1. Preferred term label: abnormal FO-wave shape

>2. Synonyms: n/a

>3. Textual definition: any anomaly in the graphical representation of the FO-wave in the electroretinogram; the ERG FO-wave can be used to assess the functional integrity of the photoreceptors, the pigment epithelial cells and the interactions between them.

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

abnormal electroretinogram waveform feature (MP:0012029)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

**Comment by sbello** [comment]
Trying to find a reference for the FO-wave definition

**Comment by sbello** [comment]
Reference from Abby PMID: 30040236 https://pubmed.ncbi.nlm.nih.gov/30040236/

**Comment by sbello** [comment]
MP:0020974
